readme.md

```diff
@@ -87,6 +87,7 @@ Click the type names for complete docs.
 - [`Entries`](source/entries.d.ts) - Create a type that represents the type of the entries of a collection.
 - [`SetReturnType`](source/set-return-type.d.ts) - Create a function type with a return type of your choice and the same parameters as the given function type.
 - [`Asyncify`](source/asyncify.d.ts) - Create an async version of the given function type.
+- [`TakeLast`](source/take-last.d.ts) - Extracts the type of the last element of an array.
```

Comment:
Since it only works on arrays, I think this name is too generic.
```suggestion
- [`LastArrayElement`](source/take-last.d.ts) - Extracts the type of the last element of an array.
```

?

Reply:
Applied.